src/libs/OptionsListUtils/index.ts

```diff
@@ -1179,7 +1180,7 @@ function processReport(
         reportMapEntry,
         reportOption: {
             item: report,
-            ...createOption(accountIDs, personalDetails, report, undefined, reportAttributesDerived),
+            ...createOption(accountIDs, personalDetails, report, {}, reportAttributesDerived),
```

Comment:
I don't think we need this change. When `undefined` is passed, the config will default to `{}`.